.github/workflows/build.yml

```diff
@@ -108,8 +133,12 @@ jobs:
           flutter pub get
       - name: Install dependencies with yarn
         run: yarn install --frozen-lockfile
+      - name: Download built binaries from build job
+        uses: actions/download-artifact@v4
+        with:
+          name: ${{ github.sha }}
       - name: Run End-to-End Tests
-        run: yarn test:e2e ${{ matrix.wizard }}
+        run: yarn test:e2e:bin ${{ matrix.wizard }}
```

Comment:
wait, does this mean we only test against the self-contained binaries but no longer with Node (i.e. simulate an `npx @sentry/wizard` run)? If so, let's please do both. Might be sense to do this in a matrix or parallelize it in some way (happy to leave details up to you)

Reply:
We never simulate `npx` properly anyway as the old way is running `node dist/bin.js` which is not entirely the same thing.

The argument here is: fossilized binaries are created from `dist/bin.js` so if fossilized works, `dist/bin.js` should definitely work. The other way around is not guaranteed as Node SEA brings in more restrictions (such as no `require`).

I can create a test matrix but I'd say it will be a waste of time and resources.

Reply:
got it. I'm fine with this for now, we can revisit when there's a need to test them separately. Thanks for explaining